packages/flutter/lib/src/gestures/pointer_signal_resolver.dart

```diff
@@ -96,6 +96,9 @@ class PointerSignalResolver {
   void resolve(PointerSignalEvent event) {
     if (_firstRegisteredCallback == null) {
       assert(_currentEvent == null);
+      event.respond(
```

Comment:
Maybe leave a comment here to explain this to future readers. Something like: Nobody in the framework/app handled the signal, allow the platform to trigger any default actions.